src/jdk.crypto.mscapi/windows/classes/sun/security/mscapi/CKeyStore.java

```diff
@@ -867,6 +885,13 @@ private int getLocation() {
         return storeLocation;
     }
 
+    /**
+     * Returns whether the keystore comes from a stream.
+     */
+    private boolean getRequiresKeyStoreFromStream() {
```

Comment:
I've followed the existing style for get[Property] here the property is RequiresKeyStoreFromStream.  Perhaps the property could be better named, but in this case it means "this KeyStore must be provided a stream from the user", the false case is missleading I guess, ie 'does not require ...' when in fact it is really "must not provide stream"

again in its final form I believe most of this will change to suit the new API